miniapps/nurbs/CMakeLists.txt

```diff
@@ -13,6 +13,18 @@ add_mfem_miniapp(nurbs_ex1
   MAIN nurbs_ex1.cpp
   LIBRARIES mfem)
 
+add_mfem_miniapp(nurbs_ex3
```

Comment:
Also add the new files to `makefile`.

Reply:
In have fixed the makefile. But introduced a bug... or somewhere in the fixes above...